lms/templates/fields/field_textarea.underscore

```diff
@@ -27,7 +27,7 @@
             <% } %>
             ><%- value %></span>
         <% } else { %>
-            ><a href="#"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></a>
+            ><button type="button"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></button>
```

Comment:
Keyboard functionality works here, despite it not being a button. Making this a button would require extensive reworking of the `fields` and which would have larger repercussions since they are templates. I suggest tackling this work in [TNL-5283](https://openedx.atlassian.net/browse/TNL-5283).